test/python/test_ana_client.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from common.mayastor import mayastor_mod
+from common.nvme import (
+    nvme_connect,
+    nvme_disconnect,
+    nvme_disconnect_all,
+    nvme_list_subsystems,
+    identify_namespace,
+)
+import uuid
+import mayastor_pb2 as pb
+import os
+
+
+POOL_NAME = "pool1"
+NEXUS_GUID = "afebdeb9-ff44-1111-2222-254f810ba34a"
+
+
+@pytest.fixture
+def create_replicas(mayastor_mod):
+    ms0 = mayastor_mod.get("ms0")
+    ms1 = mayastor_mod.get("ms1")
+
+    replicas = []
+
+    for m in (ms0, ms1):
+        p = m.pool_create(POOL_NAME, "malloc:///disk0?size_mb=100")
+        assert p.state == pb.POOL_ONLINE
+        r = m.replica_create(POOL_NAME, str(uuid.uuid4()), 32 * 1024 * 1024)
+        replicas.append(r.uri)
+
+    yield replicas
+
+    for m in (ms0, ms1):
+        try:
+            m.pool_destroy(POOL_NAME)
+        except Exception:
+            pass
+
+
+@pytest.fixture
+def create_nexuses(mayastor_mod, create_replicas):
+    uris = []
+
+    nvme_disconnect_all()
+
+    for n in ["ms2", "ms3"]:
+        ms = mayastor_mod.get(n)
+        ms.nexus_create(NEXUS_GUID, 32 * 1024 * 1024, create_replicas)
+        uri = ms.nexus_publish(NEXUS_GUID)
+        uris.append(uri)
+
+    yield uris
+
+    nvme_disconnect_all()
+
+    for n in ["ms2", "ms3"]:
+        ms = mayastor_mod.get(n)
+        ms.nexus_destroy(NEXUS_GUID)
```

Comment:
just write:
`for instance in ["ms2", "ms3]:
    mayastor_mod[instance].nexus_destroy(NEXUS_GUUID)`